contrib_bots/bots/wikipedia/wikipedia.py

```diff
@@ -31,7 +31,10 @@ def handle_message(self, message, client, state_handler):
         client.send_reply(message, bot_response)
 
     def get_bot_wiki_response(self, message, client):
+        help_text = 'Please enter your message after @mention-bot'
```

Comment:
@ mentioning a bot in pm works the same as not mentioning it at all in pm but if you want i can add a check and respond differently